packages/api/src/hooks/useCurrencyConfig.ts

```diff
@@ -63,6 +63,13 @@ const useCurrencyConfig = () => {
                 code: currency,
                 /** Currency display code */
                 display_code: currency === 'UST' ? 'USDT' : currency,
+                /** Format the balance */
+                formatBalance: (amount: number) => {
+                    return new Intl.NumberFormat('en-US', {
+                        minimumFractionDigits: currency_config.fractional_digits,
+                        maximumFractionDigits: currency_config.fractional_digits,
+                    }).format(amount);
+                },
```

Comment:
@thisyahlen-deriv I think we can move this to `useAccountsList` inside `modified_accounts_with_balance` 🙏🏻 

<img width="1561" alt="image" src="https://github.com/binary-com/deriv-app/assets/72082844/d637801d-3e52-488d-8497-1e10ed4e7c92">